app/src/main/res/layout/activity_main.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<org.wikipedia.views.LinearLayoutTouchIntercept xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
+    android:id="@+id/main_container"
```

Comment:
Would it be better if move `android:id` to the top of `android:layout_width`?

Reply:
Sure!